src/contract/polls.cpp

```diff
@@ -20,6 +20,17 @@ CBlockIndex* GetHistoricalMagnitude(std::string cpid);
 StructCPID GetLifetimeCPID(const std::string& cpid, const std::string& sFrom);
 bool WalletOutOfSync();
 
+std::string GetShareType(double dShareType)
```

Comment:
Can we change this to:
- Accept an integer
- Compare against a poll share type enum
- Drop the final cout

Reply:
Isn't the enum dangerous, because the poll published can be of a share type value that is not enumerated? Seems like this would require more checks when processing polls than currently.